#2635 Add ability to perform click at disabled elements (#2636)
selenide/selenide: CHANGELOG.md

* #2550 Implement downloading files via CDP or BiDi (#2567) - thanks to Sergey Brit!
* #2612 don't set page load timeout in mobile tests (#2628)
* #2617 User can safely add the same proxy filter many times (#2630)

> * #2635 Add ability to perform click at disabled elements (#2636)

Reviewer: ```suggestion
* #2635 Add the ability to skip checks before clicking an element (#2636)
```
